tests/fixtures/markdown/export/expected.md

```diff
@@ -0,0 +1,206 @@
+<td>timestamp</td>
+<td><i>The business timestamp in UTC when the order was successfully registered in the source system and the payment was successful.</i>
+<li><code>required</code></li>
+<li><b>tags:</b> ['business-timestamp']</li>
+<li><b>examples:</b> ['2024-09-09T08:30:00Z']</li></td>
+</tr>
+<tr>
+<td> order_total</td>
+<td>long</td>
+<td><i>Total amount the smallest monetary unit (e.g., cents).</i>
+<li><code>required</code></li>
+<li><b>examples:</b> [9999]</li></td>
+</tr>
+<tr>
+<td> customer_id</td>
+<td>text</td>
+<td><i>Unique identifier for the customer.</i>
+<li><b>minLength:</b> 10</li>
+<li><b>maxLength:</b> 20</li></td>
+</tr>
+<tr>
+<td> customer_email_address</td>
+<td>text</td>
+<td><i>The email address, as entered by the customer.</i>
+<li><b>format:</b> email</li>
+<li><code>required</code></li>
+<li><code>pii</code></li>
+<li><b>classification:</b> sensitive</li>
+<li><b>quality:</b> [{'type': 'text', 'description': 'The email address is not verified and may be invalid.'}]</li>
+<li><b>lineage:</b> {'inputFields': [{'namespace': 'com.example.service.checkout', 'name': 'checkout_db.orders', 'field': 'email_address'}]}</li></td>
+</tr>
+<tr>
+<td> processed_timestamp</td>
+<td>timestamp</td>
+<td><i>The timestamp when the record was processed by the data platform.</i>
+<li><code>required</code></li>
+<li><b>config:</b> {'jsonType': 'string', 'jsonFormat': 'date-time'}</li></td>
+</tr>
+</table>
+
+<table>
+<tr>
+<th colspan=3>LINE_ITEMS<br/>
+<i>A single article that is part of an order.</i></th>
+</tr>
+<tr>
+<td> line_item_id</td>
+<td>text</td>
+<td><i>Primary key of the lines_item_id table</i>
+<li><code>required</code></li></td>
+</tr>
+<tr>
+<td> order_id</td>
+<td>None</td>
+<td><i>No description.</i>
+<li><b>ref:</b> #/definitions/order_id</li>
+<li><b>references:</b> orders.order_id</li></td>
+</tr>
+<tr>
+<td> sku</td>
+<td>None</td>
+<td><i>The purchased article number</i>
+<li><b>ref:</b> #/definitions/sku</li></td>
+</tr>
+</table>
+
+
+## Definitions
+<table>
+<tr>
+<th>Name</th>
+<th>Type</th>
+<th>Domain</th>
+<th>Attributes</th>
+</tr>
+<tr>
+<td>order_id</td>
+<td>text</td>
+<td></td>
+<td>
+<i>An internal ID that identifies an order in the online shop.</i>
+<li><b>title:</b> Order ID</li>
+<li><b>format:</b> uuid</li>
+<li><code>pii</code></li>
+<li><b>classification:</b> restricted</li>
+<li><b>tags:</b> ['orders']</li>
+<li><b>examples:</b> ['243c25e5-a081-43a9-aeab-6d5d5b6cb5e2']</li>
+</td>
+</tr>
+<tr>
+<td>sku</td>
+<td>text</td>
+<td></td>
+<td>
+<i>A Stock Keeping Unit (SKU) is an internal unique identifier for an article. 
+It is typically associated with an article's barcode, such as the EAN/GTIN.
+</i>
+<li><b>title:</b> Stock Keeping Unit</li>
+<li><b>pattern:</b> ^[A-Za-z0-9]{8,14}$</li>
+<li><b>tags:</b> ['inventory']</li>
+<li><b>links:</b> {'wikipedia': 'https://en.wikipedia.org/wiki/Stock_keeping_unit'}</li>
+<li><b>examples:</b> ['96385074']</li>
+</td>
+</tr>
+</table>
```

Comment:
Tables also have a special markup in markdown. We should use that.

Reply:
I removed html tags except `<br>` to go to the line in table cell